### PR TITLE
[IMP] l10n_in_edi*: remove ensure_one and unused params

### PR DESCRIPTION
**Pull request by hamo-odoo**

### CODE REVIEW

File: addons/l10n_in_edi/models/account_edi_format.py [hunk 1]
```diff
@@ -238,7 +238,6 @@ def _l10n_in_edi_cancel_invoice(self, invoice):
             return {invoice: {"success": True, "attachment": attachment}}
 
     def _l10n_in_validate_partner(self, partner, is_company=False):
```
Comment (hamo-odoo) [inline addons/l10n_in_edi/models/account_edi_format.py:240]:
```suggestion
    def _l10n_in_validate_partner(self, partner):
```
Not sure why using `is_company` ? Can't see any use in the method :thinking:

Comment (jco-odoo) [reply addons/l10n_in_edi/models/account_edi_format.py:240]: Indeed, but better to do such things in master.

**Review by hamo-odoo — COMMENTED** [review]
@jco-odoo I have removed the unused parameters. LGTM

File: addons/l10n_in_edi/models/account_edi_format.py (hunk 1 above)
Comment (hamo-odoo) [reply addons/l10n_in_edi/models/account_edi_format.py]: Too late to see this comment

**Comment by hamo-odoo** [comment]
Now we have ENT PR - https://github.com/odoo/enterprise/pull/60307 too :new_moon_with_face:

**Pull request closed without merging**